packages/taquito-michel-codec/test/helpers/global-constants-helpers.ts

```diff
@@ -0,0 +1,65 @@
+import { Expr } from '../../src/micheline';
+
```

Comment:
very minor nitpick, but what do you think about renaming the file to `global-constants-helper.ts` instead for consistency. I think generally file types in names tend to be singular instead of plurals

Reply:
Thanks, I made the change.